packages/components/src/components/alert/alert.css

```diff
@@ -1,38 +1,85 @@
 .alert {
-	box-sizing: border-box;
-	background: #eee;
-	color: #333;
-	width: 100%;
-	padding: 1rem;
-	text-align: left;
+  box-sizing: border-box;
+  background: #eee;
+  color: #333;
+  width: 100%;
+  padding: 1rem;
+  text-align: left;
+  position: relative;
 }
 
 .alert--variant-primary {
-	background: blue;
-	color: #fff;
+  background: blue;
+  color: #fff;
 }
 
 .alert--variant-secondary {
-	background: #eee;
-	color: #333;
+  background: #eee;
+  color: #333;
 }
 
 .alert--variant-variant-warning {
-	background: orange;
-	color: #fff;
+  background: orange;
+  color: #fff;
 }
 
 .alert--variant-danger {
-	background: red;
-	color: #fff;
+  background: red;
+  color: #fff;
 }
 
 .alert--variant-success {
-	background: green;
-	color: #fff;
+  background: green;
+  color: #fff;
 }
 
 .alert--variant-info {
-	background: lightblue;
-	color: #fff;
+  background: lightblue;
+  color: #fff;
 }
+
+.alert--title {
+  font-size: 1rem;
+  color: white;
+  margin: 0;
+  text-align: left;
+}
+
+.alert--icon {
+  font-weight: normal;
+  color: black;
+  margin: 3px;
+  left: 0.75rem;
+  z-index: 10;
+  padding: 1px;
+  border-radius: 3px;
+  text-align: center;
+}
+
+.close {
```

Comment:
Please consider renaming `.close` to `.alert__close`

Reply:
ok